src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ManagedGroupService.scala

```diff
@@ -33,7 +35,7 @@ class ManagedGroupService(
       samUser: SamUser,
       accessInstructionsOpt: Option[String] = None,
       samRequestContext: SamRequestContext
-  ): IO[Resource] = {
+  ): IO[Resource] = openTelemetry.time("api.v1.managedGroup.create.time", API_TIMING_DURATION_BUCKET) {
```

Comment:
do u need a new duration bucket? I'm guessing group sync will take much longer than api response time (assuming `API_TIMING_DURATION_BUCKET` was for API response time)

Reply:
I was sort of hoping one bucket could be used for all the calls, I tried to do a pretty large range

Reply:
feels weird to me that one bucket fits all....but if things look good once u chart them, I'm fine with it